includes/common-link-callmethods.md

```diff
@@ -1 +1 @@
-- **Call Methods**: [Angular](/concepts/40%20Angular%20Components/20%20Component%20Configuration%20Syntax/42%20Call%20Methods.md '/Documentation/Guide/Angular_Components/Component_Configuration_Syntax/#Call_Methods') | [Vue](/concepts/55%20Vue%20Components/20%20Component%20Configuration%20Syntax/42%20Call%20Methods.md '/Documentation/Guide/Vue_Components/Component_Configuration_Syntax/#Call_Methods') | [React](/concepts/50%20React%20Components/40%20Component%20Configuration%20Syntax/60%20Call%20Methods.md '/Documentation/Guide/React_Components/Component_Configuration_Syntax/#Call_Methods') | [jQuery](/concepts/58%20jQuery%20Components/20%20Component%20Configuration%20Syntax/10%20Call%20Methods.md '/Documentation/Guide/jQuery_Components/Component_Configuration_Syntax/#Call_Methods') | [AngularJS](/concepts/Getting%20Started/Widget%20Basics%20-%20AngularJS/10%20Call%20Methods.md '/Documentation/Guide/Getting_Started/Widget_Basics_-_AngularJS/Call_Methods/') | [Knockout](/concepts/Getting%20Started/Widget%20Basics%20-%20Knockout/10%20Call%20Methods.md '/Documentation/Guide/Getting_Started/Widget_Basics_-_Knockout/Call_Methods/')
\ No newline at end of file
+- **Call Methods**: [Angular](/concepts/40%20Angular%20Components/20%20Component%20Configuration%20Syntax/42%20Call%20Methods.md '/Documentation/Guide/Angular_Components/Component_Configuration_Syntax/#Call_Methods') | [Vue](/concepts/55%20Vue%20Components/20%20Component%20Configuration%20Syntax/42%20Call%20Methods.md '/Documentation/Guide/Vue_Components/Component_Configuration_Syntax/#Call_Methods') | [React](/concepts/50%20React%20Components/40%20Component%20Configuration%20Syntax/60%20Call%20Methods.md '/Documentation/Guide/React_Components/Component_Configuration_Syntax/#Call_Methods') | [jQuery](/concepts/58%20jQuery%20Components/20%20Component%20Configuration%20Syntax/10%20Call%20Methods.md '/Documentation/Guide/jQuery_Components/Component_Configuration_Syntax/#Call_Methods') 
```

Comment:
Leave this file as it is, do not change it. It's already [redone](https://github.com/vladaskorohodova/devextreme-documentation/blob/new-navigation-tree-23.1/includes/common-link-callmethods.md?plain=1) on a new website. Please always check the original link from the new website in the card to see if the changes are made.